### DISCUSSION TITLE
Can I Output Convergence Time?

### DISCUSSION THREAD

**Discussion by gnssLa** [groops-devs/groops#191] [Q&A]
Dear Groops Developers,

I have noticed that GROOPS utilizes Robust Least Squares, which effectively downweights observations containing significant outliers beyond a specific threshold. Considering its structure and design, I am wondering if there is a way to leverage this functionality to obtain the convergence time as an output? **Or is the assumption that all computations converge when at a particular threshold, e.g: 0.01 m?**

Thank you in advance for your assistance.

Best regards,

Duarte

**Comment by tmayerguerr** [comment]
Hi Duarte,
Sorry, I'm not sure how the question is meant.  It is not easy to determine the convergence. As a criterion, one can define that the parameters may no longer change above a certain threshold (all values in relation to meter, clock parameters are multiplied by the speed of light, for example). In practice, however, this does not work well yet, so we have simply set a certain number of iterations in our processing.
regards
Torsten
